### PR TITLE
Middle section of mobile version

### PR DESCRIPTION
**Pull request by MAleemH**
I have completed middle section. Added "work section" and "about myself" section.

### CODE REVIEW

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @MAleemH,

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- All linters passed ✅
- Commit messages ✔️ 
- Reame file ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -33,12 +35,171 @@ <h1 class="header">
...
+    <!-- "Works" section -->
+    <div class="works-section">
+
+      <div class="grid-container">
+
+        <!-- item 1: tonic -->
+        <div class="grid-item">
+
+          <img src="images/portfolio_1.png" alt="Tonic">
+          <h1 class="project-title">Tonic</h1>
```
[the remark this answers is not in the document]
Comment (MAleemH) [reply index.html:53]: Please make this point clear. I'm unable to understand. Thanks

File: index.html [hunk 2]
```diff
@@ -33,12 +35,171 @@ <h1 class="header">
...
+    </div>
+    <!-- "Works" section -->
+    <div class="works-section">
+
+      <div class="grid-container">
+
+        <!-- item 1: tonic -->
+        <div class="grid-item">
+
+          <img src="images/portfolio_1.png" alt="Tonic">
```
[the remark this answers is not in the document]
Comment (MAleemH) [reply index.html:52]: Please make this point clear. I'm unable to understand. Thanks

[the remark this answers is not in the document]
Comment (Olamarx) [reply index.html:52]: I edited it, I made a mistake. I am sorry. Let me know if you still don't understand.

File: index.html [hunk 3]
```diff
@@ -33,12 +35,171 @@ <h1 class="header">
...
+
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="languages">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <a href="" class="button btn-border">See Project</a>
+
+        </div>
```
Comment (roshan-bajgain) [inline index.html:118]: - [ ] I would suggest you please stick to the design as much as possible like color, font, size, etc.
- [ ] Please increase the font size of the `h2` tag So it looks like the design.
- [ ] Please use the same border for the buttons as it is different from the Figma design
- [ ] Please fix the spacing by looking at the Figma design.
- [ ] I would suggest you please do this for every card.
### Your design
 
your
### Figma design
figma

**Review by roshan-bajgain — APPROVED** [review]
Hi @MAleemH, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Optional suggestions
__Check the comment under the review for optional suggestion__

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**